src/lib/Input.svelte

```diff
@@ -45,9 +41,20 @@
         innerTextContainsLetters();
 
     const handleKeyDown = (event: KeyboardEvent) => {
-        if (!canSend || event.key !== "Enter" || event.shiftKey) return;
-        send();
-        event.preventDefault();
+        let pressedEnter: boolean = event.key === "Enter";
+        let pressedShift: boolean = event.shiftKey;
+
+        if (pressedEnter) {
+            if (pressedShift) {
+                return;
+            }
+
+            if (canSend) {
+                send();
+            }
+
+            event.preventDefault();
```

Comment:
[Need info] was it necessary to always 'prevent default' when enter was clicked?